### PR TITLE
Add FCGI2

### PR DESCRIPTION
**Pull request by hobu**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by hobu** [comment]
@conda-forge/help-c-cpp This is ready to merge up. Thanks!

File: recipes/fcgi2/bld.bat [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+
+
+nmake /f Makefile.nt
+if errorlevel 1 exit 1
+
+copy cgi-fcgi\Release\cgi-fcgi.exe %PREFIX%\bin
```
Comment (xhochy) [inline recipes/fcgi2/bld.bat:7]:
```suggestion
copy cgi-fcgi\Release\cgi-fcgi.exe %PREFIX%\bin
if errorlevel 1 exit 1
```

File: recipes/fcgi2/bld.bat [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+copy libfcgi\Release\libfcgi.dll %PREFIX%\bin
```
Comment (xhochy) [inline recipes/fcgi2/bld.bat:8]: if errorlevel 1 exit 1

**Review by xhochy — CHANGES_REQUESTED** [review]
It would be nice if you would remove some of the duplicate blank lines in the build scripts.

File: recipes/fcgi2/bld.bat (hunk 2 above)
Comment (hobu) [reply recipes/fcgi2/bld.bat:6]: Updated as suggested in 780fc05cdec3954e4e1db11e4b684619a39e3058

File: recipes/fcgi2/bld.bat (hunk 1 above)
Comment (hobu) [reply recipes/fcgi2/bld.bat:4]: Updated as suggested in 780fc05cdec3954e4e1db11e4b684619a39e3058

**Review by xhochy — APPROVED** [review]

**Pull request merged**